### PR TITLE
[Enhancement] Write fatal log to stderr together with crash stack trace

### PR DESCRIPTION
**Pull request by decster**
This PR changes glog config to output fatal log to stderr, so when the user get a crash stacktrace caused by FATAL log, he can provide stacktrace together with actual FATAL log to the helping person, saving an extra round of communication.

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: be/src/common/logconfig.cpp [hunk 1]
```diff
@@ -143,7 +143,7 @@ bool init_glog(const char* basename, bool install_signal_handler) {
     }
 
     // Don't log to stderr.
-    FLAGS_stderrthreshold = 5;
+    FLAGS_stderrthreshold = 3;
```
Comment (sduzh) [inline be/src/common/logconfig.cpp:146]: Need update the comment

Comment (decster) [reply be/src/common/logconfig.cpp:146]: changed

**Review by sduzh — APPROVED** [review]

**Review by sevev — APPROVED** [review]

**Review by trueeyu — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.2

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/common/logconfig.cpp|1|1|100.00%|[]|

**Pull request merged**